src/types.ts

```diff
@@ -504,18 +504,59 @@ export interface FluffyStream {
   subtitleType: string;
 }
 
-export interface Metadata {
+export interface Metadata extends partialMetadata {
   show: string;
-  firstBroadcastedAt: null;
-  broadcasterShowId: null;
+}
+
+export interface partialMetadata {
```

Comment:
C'est normal le "p" minuscule?
```suggestion
export interface PartialMetadata {
```

Reply:
C'est un oubli ça par contre